skt/runner.py

```diff
@@ -397,7 +400,7 @@ def __watchloop(self):
                                       ' rest of running jobs and aborting!')
                         self.failures[cid] = [[root.attrib.get('system')],
                                               set([(result, status)]),
-                                              1]
+                                              SKT_FAIL]
```

Comment:
Again here, the last element of the list represents number of failed test runs and not the return code. Yes, we need to clean up the runner :)